aws/resource_aws_backup_plan_test.go

```diff
@@ -625,3 +760,109 @@ resource "aws_backup_plan" "test" {
 }
 `, rName)
 }
+
+func testAccAwsBackupPlanConfig_copyAction(rName1 string, rName2 string) string {
```

Comment:
Nit: Can remove the second randomized name parameter by appending `-2` or similar in the template below, e.g. `name = "%[1]s-2"`